src/Pokemon.java

```diff
@@ -0,0 +1,138 @@
+import java.util.Arrays;
+import java.util.List;
+
+ public abstract class Pokemon {
+    protected String type;
+    protected String name;
+    protected int level = 1;
+    protected int hp = 10;
+    protected String sound;
+
+    // Constructor //
+    public Pokemon (String type) {
+        this.type = type;
+    }
+
+     public Pokemon(String name, int level, int hp, String food, String sound) {
```

Comment:
level en hp hebben al een standaard implementatie gekregen, is het dan nog nodig om dit in je constructor te zetten?